build.gradle

```diff
@@ -0,0 +1,111 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
+    }
+}
+
+def versions = [
+        springBootVersion: '1.5.2.RELEASE'
+]
+
+allprojects {
+    group 'uk.gov.hmcts.reform'
+    version '0.1.0'
+
+    apply plugin: 'java'
+    apply plugin: 'java-library'
+    apply plugin: 'maven'
+    apply plugin: 'maven-publish'
+    apply plugin: 'checkstyle'
+    apply plugin: 'com.jfrog.bintray'
+    apply plugin: 'jacoco'
+
+
+    checkstyle.toolVersion = '7.2'
+    checkstyle.configFile = new File(rootDir, "checkstyle.xml")
+
+    // make build fail on Checkstyle issues (https://github.com/gradle/gradle/issues/881)
+    tasks.withType(Checkstyle).each { checkstyleTask ->
+        checkstyleTask.doLast {
+            reports.all { report ->
+                def outputFile = report.destination
+                if (outputFile.exists() && outputFile.text.contains("<error ")) {
+                  //  throw new GradleException("There were checkstyle warnings! For more info check $outputFile")
+                }
+            }
+        }
+    }
+
+    sourceCompatibility = 1.8
+
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBootVersion
+    }
+}
+
+//jacocoTestReport {
+//    reports {
+//        csv.enabled false
+//        html.enabled true
+//        // html.destination file("${buildDir}/jacocoHtml")
+//    }
+//}
```

Comment:
should this be commented out?

Reply:
Should be deleted. Will do

Reply:
will `./gradlew clean test jacocoTestReport` from travis file work without it?

Reply:
yes - the default is to produce html report